src/components/Card.jsx

```diff
@@ -8,7 +13,37 @@ export default function Card({ chatName, chatLink, from, to, dateFrom, dateTo, m
         redirectWindow.focus();
     }
 
-    console.log(dateFrom)
+    dayjs.extend(customParseFormat);
+    dayjs.extend(localizedFormat);
+    function formatDate(stringDate) {
+        if (stringDate === null) return null;
+        const date = dayjs(stringDate, 'DD-MM-YYYY');
+        return date.locale('ru').format('D MMMM YYYY');
+    }
+
+    function rewardInfo() {
+        let color;
+        let title;
+        switch (isRewardable) {
```

Comment:
You can move this condition to config file if you want and make text as locale pls check how it works 
locales.ru.rewardable
smth like that